design/account.go

```diff
@@ -109,6 +130,24 @@ var _ = a.Resource("users", func() {
 		a.Response(d.BadRequest, JSONAPIErrors)
 	})
 
+	a.Action("createUserAsServiceAccount", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.POST("/:id"),
```

Comment:
What ID are you requesting here? It should be just `POST /users`, shouldn't it?

Reply:
a076d63

Initially, I had designed it to accept the identity uuid and the user uuid ( in the payload ) but then I realized that the registration app shouldn't be burdened with the task of generating IDs. It should simply be passing on the user/identity data - and auth service should decide the IDs. 

Removed userID from payload and the identityID from the POST url as suggested you.